_data/features.json

```diff
@@ -0,0 +1,29 @@
+[
+    {
+      "title": "Create Efficiency",
+      "description": "We believe that efficiency promotes productivity.",
```

Comment:
this doesn't make much sense. if the objective here is Productivity, perhaps something like `title: “Productivity"`, and `description: "We adapt conventions and automation to promote productivity"`

Reply:
see https://eqworks.slack.com/archives/C02MNBJA33N/p1637619614033900 for detailed discussions